--- configure-data-access.sh.orig       2025-07-11 17:06:49.533548225 +0000
+++ configure-data-access.sh    2025-07-11 17:07:07.873550949 +0000
@@ -13,6 +13,19 @@
 ask "S3_ACCESS_KEY" "Enter the S3 (MinIO) access key" "" is_non_empty
 ask "S3_SECRET_KEY" "Enter the S3 (MinIO) secret key" "" is_non_empty
 
+ask "EOAPI_POSTGRES_CLUSTER" "Install Postgress database in the Kubernetes cluster" "yes" is_yes_no
+if [[ "${POSTGRES_CLUSTER}" == "no" ]]; then
+  ask "EOAPI_POSTGRES_HOST" "Enter the URL of the Postgres database host" "" is_valid_domain
+  ask "EOAPI_POSTGRES_PORT" "Enter the port of the Postgres database host" "5432" is_non_empty
+  ask "EOAPI_POSTGRES_DB" "Enter the database name" "eoapi" is_non_empty
+  ask "EOAPI_POSTGRES_USER" "Enter the database user" "" is_non_empty
+  ask "EOAPI_POSTGRES_PASS" "Enter the database password" "" is_non_empty
+fi
+
+ask "EOAPI_COMPONENT_STAC" "Enable STAC" "true" is_boolean
+ask "EOAPI_COMPONENT_RASTER" "Enable Raster (TiTiler)" "true" is_boolean
+ask "EOAPI_COMPONENT_VECTOR" "Enable Vector (TiPg)" "true" is_boolean
+ask "EOAPI_COMPOPNENT_MULTIDIM" "Enable Multidim" "true" is_boolean
 
 gomplate  -f "eoapi/$TEMPLATE_PATH" -o "eoapi/$OUTPUT_PATH" --datasource annotations="$GOMPLATE_DATASOURCE_ANNOTATIONS"
 gomplate  -f "postgres/$TEMPLATE_PATH" -o "postgres/$OUTPUT_PATH" --datasource annotations="$GOMPLATE_DATASOURCE_ANNOTATIONS" 